agent/xds/testdata/clusters/connect-proxy-with-chain-and-splitter.latest.golden

```diff
@@ -49,6 +49,54 @@
       },
       "type": "EDS"
     },
+    {
+      "@type": "type.googleapis.com/envoy.config.cluster.v3.Cluster",
+      "altStatName": "db.default.dc1.internal.11111111-2222-3333-4444-555555555555.consul",
+      "circuitBreakers": {},
+      "commonLbConfig": {
+        "healthyPanicThreshold": {}
+      },
+      "connectTimeout": "25s",
```

Comment:
This line shows that the service resolver connect timeout works

Reply:
This db cluster was added because it has been added as a destination on the splitter